### PR TITLE
Feature/david/snackbar updates

### PR DESCRIPTION
**Pull request by malmstein**
https://app.asana.com/0/414730916066338/1199702927327729/f
Adds missing tests for https://github.com/duckduckgo/Android/pull/951

**Screenshots**:
| Fireproof | Bookmarks |
| ------ | ----- | 
device-2021-02-05-135947|device-2021-02-05-135931|

### CODE REVIEW

**Review by cmonfortep — COMMENTED** [review]
Everything looks great. Just a few comments from my side about the undo functionality:
- the position of the element in the list changes
- the favicon is lost
- we should validate the new copy with the team ( in https://github.com/duckduckgo/Android/issues/945 the proposed copy was different)

What do you think? maybe the latter is more important. As for the rest, I will leave it up to you, no strong opinions from my side since this is just a nice to have functionality.

**Comment by malmstein** [comment]
Thanks @cmonfortep !

On your comments:
- Position is reordered, I don't see this is a problem
- Favicon is not lost in any of my tests, let's sync and see what are we doing different
- Agree, I've opened a task to get this reviewed

**Comment by malmstein** [comment]
@cmonfortep PR updated with the Copy reviewed

File: versions.properties [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## suppress inspection "SpellCheckingInspection" for whole file
```
Comment (cmonfortep) [inline versions.properties:1]: 👀

**Review by cmonfortep — APPROVED** [review]
LGTM, sorry for the late review. This looks great. You are right, favicon does not disappear...maybe we fixed something in between reviews.

Anyways, ship it! (but check that versions.properties file, not sure why it's there)

File: versions.properties (hunk 1 above)
Comment (malmstein) [reply versions.properties:1]: This is from `refreshVersions` plugin, part of https://github.com/duckduckgo/Android/pull/1090

**Comment by malmstein** [comment]
Thanks for the review @cmonfortep !

**Pull request merged**